inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -924,24 +1088,26 @@ for lack of consent. The reasons for no consent are shown below.
 
 This chunk will create a table of dropped individuals by reason for no consent. 
 
-Note that these reasons are spread over multiple columns, thus the proportion
-needs to be of the total and missings need to be dropped. 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 
 ```{r consent_reasons}
```

Comment:
@pbkeating this is an example where gtsummary is not quite satisfactory because the variable name shows up and is just a bit annoying. (dont worry about the name/value cleaning that will be sorted when I implement the labelling chunk above....)

Reply:
I can't see what has been done here, but looks like you already have a solution?